docs/pelion_ready/device-management-config.md

```diff
@@ -2,47 +2,47 @@
 
 - [Introduction to Mbed OS bootloaders](../porting/bootloader.html).
 - [Creating and using an Mbed OS bootloader](../tutorials/bootloader.html).
 - [Bootloader configuration in Mbed OS](../reference/bootloader-configuration.html).
-- [Mbed Bootloader for Device Management](https://github.com/ARMmbed/mbed-bootloader), or the short introduction [in the Device Management documentation](https://cloud.mbed.com/docs/current/updating-firmware/bootloaders.html).
+- [Mbed bootloader for Device Management](https://github.com/ARMmbed/mbed-bootloader), or the short introduction [in the Device Management documentation](https://www.pelion.com/docs/device-management/current/updating-firmware/bootloaders.html).
 - [Updating devices with Mbed CLI](../tools/cli-update.html).
 
-To hasten this process, you can copy the configuration from the [application example](https://github.com/ARMmbed/pelion-ready-example/blob/master/mbed_app.json) as the basis for your application configuration.
+To speed up this process, you can copy the configuration from the [application example](https://github.com/ARMmbed/pelion-ready-example/blob/master/mbed_app.json) as the basis for your application configuration.
 
-## 1. Application configuration
+## Application configuration
 
 Edit the `mbed_app.json` file, and create a new entry under `target_overrides` with the target name for your device:
 
-- **Connectivity** - Specify the default connectivity type for your target. It's essential with targets that lack default connectivity set in `targets.json` or for targets that support multiple connectivity options. For example:
+- **Connectivity** - Specify the default connectivity type for your target. It is essential with targets that lack default connectivity set in `targets.json` or for targets that support multiple connectivity options. For example:
 
    ```
             "target.network-default-interface-type" : "ETHERNET",
    ```
 
    The possible options are `ETHERNET`, `WIFI` and `CELLULAR`.
 
-   Depending on connectivity type, you might have to specify more config options. For an example, please see the defined `CELLULAR` targets in `mbed_app.json`.
+   Depending on connectivity type, you may have to specify more options. For an example, see the defined `CELLULAR` targets in `mbed_app.json`.
 
 - **Storage** - Specify the storage block device type, which dynamically adds the block device driver you specified at compile time. For example:
 
    ```
             "target.components_add" : ["SD"],
    ```
 
-   Valid options are `SD`, `SPIF`, `QSPIF` and `FLASHIAP` (not recommended). For more available options, please see the [block device components](https://github.com/ARMmbed/mbed-os/tree/master/components/storage/blockdevice).
+   Valid options are `SD`, `SPIF`, `QSPIF` and `FLASHIAP` (not recommended). For more available options, see the [block device components](https://github.com/ARMmbed/mbed-os/tree/master/components/storage/blockdevice).
```

Comment:
Please see earlier comment about "please".

Reply:
Still not that polite.